homeassistant/components/solarlog/const.py

```diff
@@ -26,14 +29,15 @@
 class SolarLogSensorEntityDescription(SensorEntityDescription):
     """Describes Solarlog sensor entity."""
 
-    factor: float | None = None
+    value: Callable[[float, int], float] | Callable[[datetime], datetime] | None = None
```

Comment:
There are never two parameters in the lambda. It's always one parameter, that can have different types.

Reply:
```suggestion
    value: Callable[[float | int], float] | Callable[[datetime], datetime] | None = None
```